### PR TITLE
fault-proof: kzg point evaluation precompile

### PR DESCRIPTION
**Pull request by Inphi**
Specify oracle interaction for KZG point evaluation precompile results.

### CODE REVIEW

File: specs/experimental/fault-proof/index.md [hunk 1]
```diff
@@ -156,7 +159,24 @@ Key: `5 ++ keccak256(commitment ++ z)[1:]`, where:
 - `commitment` is a bytes48, representing the KZG commitment.
 - `z` is a big-endian `uint256`
 
-#### Type `6-128`: reserved range
+#### Type `6`: Global EIP-4844 Point-evaluation precompile key
+
```
Comment (tynes) [inline specs/experimental/fault-proof/index.md:163]: Why is this necessary? We stub out the execution of the precompile in the program with a lookup to the oracle?

Comment (Inphi) [reply specs/experimental/fault-proof/index.md:163]: That's exactly it. Executing the precompile in the program is terribly expensive for cannon. It's much cheaper to offload this to the oracle.
This involves changes to op-geth to facilitate this in the op-program (https://github.com/ethereum-optimism/op-geth/pull/242).

Comment (tynes) [reply specs/experimental/fault-proof/index.md:163]: Would love this kind of context to be included in the specs

Comment (Inphi) [reply specs/experimental/fault-proof/index.md:163]: Good suggestion, thanks. I've added a section detailing this.

File: specs/experimental/fault-proof/index.md [hunk 2]
```diff
@@ -156,7 +159,26 @@ Key: `5 ++ keccak256(commitment ++ z)[1:]`, where:
...
+An EIP-4844 point-evaluation precompile result. It result maps directly to the EIP-4844
```
Comment (ajsutton) [inline specs/experimental/fault-proof/index.md]:
```suggestion
An EIP-4844 point-evaluation precompile result. It maps directly to the EIP-4844
```

?

File: specs/experimental/fault-proof/index.md [hunk 3]
```diff
@@ -408,6 +436,19 @@ Requests the host to prepare the L2 Output at the l2 output root `<outputroot>`.
 The L2 Output is the preimage of a
 [computed output root](../../protocol/proposals.md#l2-output-commitment-construction).
 
+### Precompile Accelerators
+
+Precompiles that are too expensive to be executed in a fault-proof VM can be executed
+more efficiently using the pre-image oracle.
+This approach ensures that the fault proof program can complete a state transition in some
```
Comment (ajsutton) [inline specs/experimental/fault-proof/index.md]:
```suggestion
This approach ensures that the fault proof program can complete a state transition in a reasonable
```
?

**Review by ajsutton — APPROVED** [review]
LGTM. Couple of suggestions but not blockers.  Probably would be good to get @tynes approval before merging as well.

**Review by tynes — APPROVED** [review]
Looks good to me! Appreciate the extra details :)

**Pull request merged**